parallax/parallax/examples/lm1b/language_model.py

```diff
@@ -30,17 +31,19 @@ def __init__(self, num_steps):
     self.num_sampled = 8192
 
   def build(self, input_shape):
-    shard_size = int((self.vocab_size + self.num_shards - 1) / self.num_shards)
-    self.emb = [self.add_variable(name='emb_%d' % i,
-                                  shape=[shard_size, self.emb_size],
-                                  initializer=tf.uniform_unit_scaling_initializer(),
-                                  trainable=True,
-                                  dtype=tf.float32) for i in range(self.num_shards)]
-    self.softmax_w = [self.add_variable(name='softmax_w_%d' % i,
-                                        shape=[shard_size, self.projected_size],
+    partitioner = parallax.get_partitioner(self.num_shards)
```

Comment:
Indentation.

Reply:
The indentation for LM1B example codes is 2 instead of 4 in Parallax's core codes so the codes are fixed for 2 indentations.